### PR TITLE
Changelog thru october

### PR DESCRIPTION
**Pull request by kevingranade**
#### Summary
SUMMARY: None

#### Purpose of change
Logs for the change god!
Turns  out there was an issue where a certain configuration of PRs would make the changelog generation script give up early and not match PRs to commits.

#### Describe the solution
Make the changelog generation script not prune un-merged PRs out of it's fetch routine. This means the script fetches more stuff, but insure it gets everything it needs.
Reverted the previously comitted July and August changelogs because some of them were tainted by this bug.
Then the usual import followed by copyediting.

### CODE REVIEW

File: data/changelog.txt [hunk 1]
```diff
@@ -193,24 +232,32 @@ Bathtubs can be used like kegs or water heaters to store liquids.
...
+Rework secubot into a TALON UGV.
+Added new human corpses types.
+Add a new M240 equipped CROWS II to outposts.
+Adds a new spell that provides invisibility.
+Adds the Hounds of Tindalos, tough Nether monsters that can appear from teleglow.
+Adds new faction camp expansion option, livestock area.
+Gender-specific clothing on corpses.
+Adds new faction expansion: storage house, includes: stone, log, wood panel, scrap metal and wattle/daub palettes.
+Add Grocery bot to carry your groceries.
+Add blackpowder loads for some cartridges
```
Comment (ZhilkinSerg) [inline data/changelog.txt]:
```suggestion
Add blackpowder loads for some cartridges.
```

File: data/changelog.txt [hunk 2]
```diff
@@ -407,37 +490,111 @@ Cloaking is now tied to a json flag, allowing mods to introduce their own cloaki
...
+simplify and improve flatpak support.
+Allow building with Clang using MinGW-w64 libs.
+Check translator comments with clang-tidy.
 
 ## I18N and A11Y:
-Use translation markers to increase performance.
-Make the translation for field name display.
+Use translation markers to increase performance
+Make the translation for field name display
+Automatically switch IME mode between text input and raw input on Windows
```
Comment (ZhilkinSerg) [inline data/changelog.txt]:
```suggestion
Automatically switch IME mode between text input and raw input on Windows.
```

File: data/changelog.txt [hunk 3]
```diff
@@ -407,37 +490,111 @@ Cloaking is now tied to a json flag, allowing mods to introduce their own cloaki
...
+Correctly extract npc dialogue lines for translation
+Allow translation of non-character key names
```
Comment (ZhilkinSerg) [inline data/changelog.txt]:
```suggestion
Allow translation of non-character key names.
```

File: data/changelog.txt [hunk 4]
```diff
@@ -407,37 +490,111 @@ Cloaking is now tied to a json flag, allowing mods to introduce their own cloaki
...
+Add position marker to some format strings
```
Comment (ZhilkinSerg) [inline data/changelog.txt]:
```suggestion
Add position marker to some format strings.
```

File: data/changelog.txt [hunk 5]
```diff
@@ -407,37 +490,111 @@ Cloaking is now tied to a json flag, allowing mods to introduce their own cloaki
...
+Support plural strings in the translation class
```
Comment (ZhilkinSerg) [inline data/changelog.txt]:
```suggestion
Support plural strings in the translation class.
```

File: data/changelog.txt [hunk 6]
```diff
@@ -407,37 +490,111 @@ Cloaking is now tied to a json flag, allowing mods to introduce their own cloaki
...
+Add table of contents to JSON_INFO.md
+Add a clang-tidy check to check for text style in the c++ code.
+Encapsulate bionic power and use setter and getter functions.
+Move player armor functions to Character scope.
 
 ## Build:
 Npctalk: add a python dialogue validator.
 Update homebrew install instructions.
-Adds Flatpak build recipe.
+Adds Flatpak build recipe/
```
Comment (ZhilkinSerg) [inline data/changelog.txt]:
```suggestion
Adds Flatpak build recipe.
```

**Comment by aoae23** [comment]
It's beautiful...

**Pull request merged**